### PR TITLE
bcompare: 4.4.4.27058 -> 4.4.6.27348 https://www.scootersoftware.com/download/v4changelog

### PR DESCRIPTION
**Pull request by kpe**
version bump for bcompare

### CODE REVIEW

**Comment by ktor** [comment]
Hi @kpe , thank you for the PR! I've tried to build it now and it works only with version = "4.4.6.27483";

File: pkgs/applications/version-management/bcompare/default.nix [hunk 1]
```diff
@@ -4,19 +4,19 @@
 
 let
   pname = "bcompare";
-  version = "4.4.4.27058";
+  version = "4.4.6.27348";
```
Comment (ktor) [inline pkgs/applications/version-management/bcompare/default.nix:7]: builds with: **version = "4.4.6.27483";**

Comment (ktor) [reply pkgs/applications/version-management/bcompare/default.nix:7]: review

**Comment by kirillrdy** [comment]
greetings @kpe, thank you for your PR

few things to resolve 
- please apply suggested changes
- squash commits, this is done to aid git bisect
- adjust commit message to follow guidelines, this is to allow automatic running of checks for pull requests

otherwise, welcome to the journey !

**Comment by ckt114** [comment]
I tried building with these changes on my local machine and the hash changed again.

**Comment by wegank** [comment]
bcompare is at 4.4.6.27483 on master, closing.

**Pull request closed without merging**